justfile

```diff
@@ -15,10 +15,10 @@ _rustflags := env_var_or_default("RUSTFLAGS", "")
 
 # If we're running in Github Actions and cargo-action-fmt is installed, then add
 # a command suffix that formats errors.
-_fmt := if env_var_or_default("GITHUB_ACTIONS", "") != "true" { "" } else {
+_fmt := if env_var_or_default("GITHUB_ACTIONS", "") != "true" { "-- -Dwarnings" } else {
```

Comment:
@hawkw is there a better way to do this I can't think of? The `-- -Dwarnings` has to be AFTER any other args but BEFORE the pipe.